.github/workflows/ci.yml

```diff
@@ -0,0 +1,322 @@
+name: LinuxKit CI
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build & Test
+    strategy:
+      matrix:
+        arch:
+          - amd64-linux
```

Comment:
I do wish that there were a way to source this from some other file, so that the "official supported matrix" would be available as a core part of the repo at the top level, and not buried deep under `.github/...`. Is that doable? Or not really?

Reply:
I don't think it's do-able as actions doesn't support YAML anchors or anything like that.

Reply:
Looks like it. Oh well, it was a nice try.